source/user-guide/creating-environments.html.md.erb

```diff
@@ -134,26 +134,26 @@ The instance-management role is a role given to assist database administrators.
 
 #### data-engineering
 
-Created for data engineers to have developer permissions as well as glue and athena permissions all in one policy. 
+Created for data engineers to have developer permissions as well as glue and athena permissions all in one policy.
 
 - Log into the console.
 - Data engineering permissions.
 
 #### administrator
 
- - Full admin access
+- Full admin access
 
 This is a legacy role and will not be generally available.
 
 ### Environment name
 
-Which environments you wish to use, we provide [core networking](creating-networking.html) for up to 4 accounts as standard.
-Usually two environments per application will be enough, a production account and a development account.
+Which environments do you wish to use, we provide [core networking](creating-networking.html) for up to 4 accounts as standard.
```

Comment:
I think the original wording was fine here - "`Which environments you wish to use`.

Reply:
agreed, reverted to the original wording.